skrf/calibration/deembedding.py

```diff
@@ -524,14 +523,14 @@ def deembed(self, ntwk):
 
         # check if the frequencies match with dummy frequencies
         if ntwk.frequency != self.short.frequency:
-            raise(ValueError('Network frequencies dont match dummy frequencies.'))
-
-        # TODO: attempt to interpolate if frequencies do not match
+            warnings.warn('Network frequencies dont match dummy frequencies, attempting overlap.', RuntimeWarning)
+            ntwk, sh = overlap_multi([ntwk, self.short])
+        else:
+            sh = self.short.copy()

```

Comment:
It can be as simple as ``sh = self.short`` similar to the open deembedding because it does not overwrite ```self.short```.

Reply:
Good catch, no reason to make a copy here